app/src/main/java/com/andrew/tasky/agenda/data/agenda/AgendaRepositoryImpl.kt

```diff
@@ -46,10 +47,21 @@ class AgendaRepositoryImpl(
                 taskEntity.toTask()
             }
         }
+        val events = db.getEventDao().getEventsOfDate(
+            startEpochMilli = startEpochMilli,
+            endEpochMilli = endEpochMilli
+        ).map {
+            it.map {
+                eventEntity ->
+                eventEntity.toEvent()
+            }
+        }
 
         emit(
             reminders.combine(tasks) { reminderFlow, taskFlow ->
                 reminderFlow + taskFlow
+            }.combine(events) { reminderTaskFlow, eventsFlow ->
+                reminderTaskFlow + eventsFlow
```

Comment:
I assume `reminderFlow` etc. are lists of agenda items right? Then why call them flow and not `reminders`?